src/utils/config.js

```diff
@@ -0,0 +1,48 @@
+const fs = require('fs');
+const os = require('os');
+const path = require('path');
+
+/**
+ * @typedef {Object} WrhsNextConfig
+ * @property {string} baseUrl Warehouse API base url
+ * @property {Object} auth Warehouse auth info
+ *
+ * @typedef {import('@oclif/dev-cli').}
+ */
+
+const DEFAULT_CONFIG_FILENAME = '.wrhs_next';
```

Comment:
Can we just use the existing config format (or some superset of it) and look for `.wrhs` instead?